guardian/VaultDoor
# Fix/project search

## What does this change?
Closes https://codemill.atlassian.net/browse/GP-356.

Fixes the Project Summary page for "new style" IDs and allows their download in the Download manager

## How to test
Get a (fully numeric) project ID from pluto and then go to https://vaultdoor.gnm.int/byproject?project={numeric-id}. Once you're on the right vault, you should see the summary for that project. Clicking the download link should allow you to download the content

## How can we measure success?
Able to download project content from InternalArchive
